build/forklift-controller/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM registry.access.redhat.com/ubi9/go-toolset:1.21.11-9 AS builder
```

Comment:
Could we do `Containerfile.controller`, wdyt @yaacov ?

Reply:
Or will we have multiple container files in the directory? Ie also for the downstream? @ernesgonzalez33

Reply:
my vote is for separate directories for each tool,
but I can also see the benefit in one directory with different names.

Reply:
I don't think you'll have different Dockerfiles in the same directory, but I find this way easier to know which one is which

Reply:
Ah last note please use the `Containerfile`

Reply:
Sure! I'll rename it!